### PR TITLE
Feat: Events page

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: packages/react-components/src/organisms/MainNavigation.tsx [hunk 1]
```diff
@@ -69,9 +78,16 @@ const MainNavigation: React.FC<MainNavigationProps> = ({
       </li>
       <li>
         <NavigationLink href={newsAndEventsHref} icon={newsIcon}>
-          News and Events
+          {isEnabled('EVENTS_PAGE') ? 'News' : 'News and Events'}
```
Comment (jeysal) [inline packages/react-components/src/organisms/MainNavigation.tsx:81]: This needs test (regression and non-regression that distinguish) - or you could ask whether changing to `News` immediately is okay

File: packages/react-components/src/organisms/__tests__/MainNavigation.test.tsx [hunk 2]
```diff
@@ -8,8 +9,22 @@ const props: ComponentProps<typeof MainNavigation> = {
...
   newsAndEventsHref: '/news-and-events',
+  eventsHref: '/events',
 };
 
+it('renders the navigation items (REGRESSION)', () => {
+  disable('EVENTS_PAGE');
+  const { getAllByRole } = render(<MainNavigation {...props} />);
+  expect(
+    getAllByRole('listitem').map(({ textContent }) => textContent),
+  ).toEqual([
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/MainNavigation.test.tsx:20]: `.not.toContain`... for less duplication

File: packages/react-components/src/organisms/__tests__/MainNavigation.test.tsx [hunk 3]
```diff
@@ -8,8 +9,22 @@ const props: ComponentProps<typeof MainNavigation> = {
   networkHref: '/network',
   sharedResearchHref: '/shared-research',
   newsAndEventsHref: '/news-and-events',
+  eventsHref: '/events',
 };
 
+it('renders the navigation items (REGRESSION)', () => {
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/MainNavigation.test.tsx:15]:
```suggestion
it('does not render a calendar navigation item (REGRESSION)', () => {
```

File: packages/react-components/src/templates/EventsCalendar.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import css from '@emotion/css';
+import { CalendarResponse } from '@asap-hub/model';
+
+import { perRem } from '../pixels';
+import { GoogleCalendar, CalendarList } from '../organisms';
+
+const containerStyles = css({
```
Comment (jeysal) [inline packages/react-components/src/templates/EventsCalendar.tsx:8]: such easy component assembly, much design system, very componentization, wow

File: packages/react-components/src/templates/EventsPage.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import css from '@emotion/css';
+
+import { perRem } from '../pixels';
+import EventsPageHeader from './EventsPageHeader';
+import { contentSidePaddingWithNavigation } from '../layout';
+
+const articleStyles = css({
+  alignSelf: 'stretch',
```
Comment (jeysal) [inline packages/react-components/src/templates/EventsPage.tsx:9]: I feel like something will need a change of defaults :D almost everywhere has this alignment

**Review by jeysal — COMMENTED** [review]
Oopsie I went on a suggesting removing or adding one letter for naming consistency spree so you can now go on an add suggestion to batch spree :nail_care:

File: packages/fixtures/src/calendars.ts [hunk 6]
```diff
@@ -0,0 +1,110 @@
+import { ListCalendarResponse, CalendarResponse } from '@asap-hub/model';
+
+const calendarResponses: CalendarResponse[] = [
```
[the remark this answers is not in the document]
Comment (fampinheiro) [reply packages/fixtures/src/calendars.ts:3]: We need to "obfuscate" them :scream:

[the remark this answers is not in the document]
Comment (kmaid) [reply packages/fixtures/src/calendars.ts:3]: obfuscated 😅

**Pull request merged**